### PR TITLE
Use original arg names if possible

### PR DESCRIPTION
**Pull request by voznesenskym**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #95898

Use graphargs

rm

cc @soumith @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

File: test/dynamo/test_export.py [hunk 1]
```diff
@@ -2009,6 +2009,23 @@ def func(x):
...
+            def __init__(self):
+                super().__init__()
+                self.my_lin = torch.nn.Linear(3, 4, bias=True)
+
+            def forward(self, x):
+                return self.my_lin(x)
+
+        mod, input_tensor = BasicModule(), torch.randn(2, 3)
+        gm, guard = torch._dynamo.export(mod, input_tensor, aten_graph=True)
+        print(gm.graph)
```
Comment (suo) [inline test/dynamo/test_export.py:2023]: stray prints

also, this should probably be testing something about the graphs input names?
And can we test the weird corner cases: `*args`, `**kwargs`, etc.

**Review by suo — APPROVED** [review]

**Comment by BowenBao** [comment]
Got a PR that tries to do the same thing lol. @voznesenskym please take a look if you haven't already. https://github.com/pytorch/pytorch/pull/95851

**Pull request closed without merging**